clang/lib/AST/CXXInheritance.cpp

```diff
@@ -170,13 +170,21 @@ bool CXXBasePaths::lookupInBases(ASTContext &Context,
     QualType BaseType =
         Context.getCanonicalType(BaseSpec.getType()).getUnqualifiedType();
 
+    bool isCurrentInstantiation = isa<InjectedClassNameType>(BaseType);
+    if (!isCurrentInstantiation) {
+      if (auto *BaseRecord = cast_or_null<CXXRecordDecl>(
```

Comment:
```suggestion
      if (auto *BaseRecord = cast_if_present<CXXRecordDecl>(
```

Can this actually be null?

Reply:
Yes, you are right. Fixed here and in the other part of the code because there is the same case.